Formula/micronaut.rb

```diff
@@ -1,17 +1,26 @@
 class Micronaut < Formula
   desc "Modern JVM-based framework for building modular microservices"
   homepage "https://micronaut.io/"
-  url "https://github.com/micronaut-projects/micronaut-core/releases/download/v1.3.6/micronaut-1.3.6.zip"
-  sha256 "271a5af14f87e78ec8c72cfd09d04c50c96826532a5496bd9651a2775815693f"
+  url "https://github.com/micronaut-projects/micronaut-starter/archive/v2.0.0.RC1.tar.gz"
+  sha256 "91dc71b200c598a91b95a20792dcc3f3cb160b8e169b581f6b98689dc713acad"
 
-  bottle :unneeded
+  GRAALVM_HOME="./graalvm/Contents/Home".freeze
 
-  depends_on "openjdk"
+  depends_on :java => ["1.8", :build]
```

Comment:
Does graal not work with Java 11?

Reply:
It does. Note that I had to put the `depends_on` clause so that I can set a custom `JAVA_HOME` variable to compile the sources.

Reply:
I'm not sure what you mean but variables can also be set when you use `depends_on "openjdk"` and we try to get rid of the old Java version dependencies.

Reply:
Without it, the audit message was:

```
Use `depends_on :java` to set JAVA_HOME
```

But I now see it also works with `openjdk`